src/Tizen.NUI/src/internal/WebView.cs

```diff
@@ -572,6 +591,31 @@ public Size ContentSize
             }
         }
 
+        /// <summary>
+        /// Gets title of web page.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string Title
+        {
+            get
+            {
+                return (string)GetValue(TitleProperty);
+            }
+        }
+
+        /// <summary>
+        /// Gets fav icon.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public ImageView Favicon
+        {
+            get
+            {
+                global::System.IntPtr imageView = Interop.WebView.GetFavicon(SwigCPtr);
```

Comment:
You need to catch exception with
```C#
if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
```
Since I`nterop.WebView.GetFavicon` is connected with `CSharp_Dali_WebView_GetFavicon` in c++ and the method can produce an error.

Reply:
Done.
A little difference is that there is a VS warning if throw an exception, null is returned instead to avoid warning.